FundacionParaguaya/MentorApp
# Draft performance

**Describe the changes**
This change improves the performance of the draft by changing the question screen. Instead of all of them being mounted at the same time, now the screen uses 'replace' instead of 'push' to unmount the previous screen and mount the one that is relevant. This reduces RAM usage as no longer 60 screens are mounted at the same time.

**To Test**
Specific steps to test when reviewing:
1. Go to Question screen
2. Add console log to componentDIdMount and componentWillUnmount
3. Make sure each time you go to the next question it mounts and the previous screen unmounts
4. Go back and forth to make sure it works in both directions.

**Related issues**
- Resolves #226
